stdlib/logging/handlers.pyi

```diff
@@ -130,31 +194,49 @@ class NTEventLogHandler(Handler):
     def getMessageID(self, record: LogRecord) -> int: ...
 
 class SMTPHandler(Handler):
-    # TODO `secure` can also be an empty tuple
+    mailhost: str  # undocumented
+    mailport: Optional[int]  # undocumented
+    username: Optional[str]  # undocumented
+    password: str  # undocumented
```

Comment:
Since you mentioned that this is only available sometimes, it would be good to add a comment about this here.

Reply:
Thanks!